.github/workflows/bundle-image-build-push.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Build and Push Bundle Image'
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+      - 'release-*'
+    tags:
+      - 'v*'
+
+jobs:
+  push-quay:
+    name: Build and Push Manifest
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+    - name: Checkout Push to Registry action
```

Comment:
Wouldn't we want the bundle step to run after the operator step to make sure, when we make a `v2.1.0-alpha.1`, that the container image exists before we bundle it?

Reply:
I don't think it will matter much for these two. My feeling is we need all the artifacts to be ready before we generate or update an index image and/or create a PR for community operators. But as far as I know there's nothing as far as the operator container and bundle images that are dependent on the others.

@shawn-hurley do you have an opinion?

Reply:
Actually I take it back. We will need the SHA, so yes, after.